daemon/cluster/executor/container/executor.go

```diff
@@ -72,27 +72,29 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 
 	// add v2 plugins
 	v2Plugins, err := e.backend.PluginManager().List(filters.NewArgs())
-	if err == nil {
-		for _, plgn := range v2Plugins {
-			for _, typ := range plgn.Config.Interface.Types {
-				if typ.Prefix != "docker" || !plgn.Enabled {
-					continue
-				}
-				plgnTyp := typ.Capability
-				switch typ.Capability {
-				case "volumedriver":
-					plgnTyp = "Volume"
-				case "networkdriver":
-					plgnTyp = "Network"
-				case "logdriver":
-					plgnTyp = "Log"
-				}
-
-				plugins[api.PluginDescription{
-					Type: plgnTyp,
-					Name: plgn.Name,
-				}] = struct{}{}
+	if err != nil {
```

Comment:
I wonder if it was done on purpose though; skip V2 plugins on failure and only use the V1 (legacy) plugins (original PR was https://github.com/moby/moby/pull/29378)